doc/2/api/essentials/handling/index.md

```diff
@@ -3,7 +3,7 @@ code: false
 type: page
```

Comment:
Same remark as for the error code page, IMHO the url will be nicer like this `api/essentials/error-handling`